libmscore/chord.cpp

```diff
@@ -1281,7 +1281,7 @@ qreal hookAdjustment(QString font, int hooks, bool up, bool small)
       {
       bool fallback = MScore::useFallbackFont && (hooks > 5);
 
-      if (font == "Emmentaler" && !fallback) {
+      if (font == "Leland" || font == "Emmentaler" && !fallback) {
```

Comment:
Not sure about the order of evaluation here, would prefere parens to make it if clearer

Reply:
good catch, I've forgotten to remove that one, TY